### DISCUSSION TITLE
Copy/clone container action

### DISCUSSION THREAD

**Discussion by rkelleyrtp** [lxdware/lxd-dashboard#51] [Q&A]
Greetings Matt,

Awesome project - really love the interface!  Fast, clean, and it ***just works***.  Kudos to you.

One action I could not find in the UI is copy/clone containers.  Maybe I missed it?  We have some "golden image" containers that are used as the baseline for other containers.  Is this feature on your roadmap?

**Comment by matthewalanpenning** [comment]
Thank you! When the container is stopped you can find copy by selecting the hamburger menu on the top right of the individual container page.
![image](https://user-images.githubusercontent.com/12444778/200458618-82e45841-2a05-4768-8ec1-00a1bcda505f.png)

**Reply by rkelleyrtp** [reply]
Thanks Matt!  I knew I over looked it somewhere.  Appreciate the fast reply...
